### PR TITLE
Documentation: SVGs hard to read in dark mode

### PR DESCRIPTION
**Pull request by maturar**
Issues: #187 

# Definition of Done

The PR shall be merged only if all items mentioned in [CONTRIBUTING.md](https://github.com/eclipse-ankaios/ankaios/blob/main/CONTRIBUTING.md#how-to-contribute) have been followed. In case an item is not applicable as described, please provide a short explanation in the description.

- [x] All steps in [CONTRIBUTING.md](https://github.com/eclipse-ankaios/ankaios/blob/main/CONTRIBUTING.md#how-to-contribute) have been handled

### CODE REVIEW

**Comment by windsource** [comment]
Using a white background is fine for me. For the image format I prefer png rather than svg. SVGs are not rendered equally on all platforms. In the past we had some problems with missing fonts which were used in SVGs but not available on certain platforms.
In addition to the white background I would also use a border of 10 (in Draw.io: File - Properties - Border Width).

**Comment by windsource** [comment]
There are still some more mermaid titles on the "Control Interface" page that are hard to read in dark mode. Not sure how that can be fixed.

**Comment by maturar** [comment]
I have investigated the topic with titles in diagrams created by mermaid. This seems to be a bug in mermaid. I found out similar bug ticket about it:
https://github.com/laurent22/joplin/issues/4939
The issue had been closed due to inactivity. There are more issues in are "mermaid and colors in the dark mode". For example:
https://forum.obsidian.md/t/mermaid-diagrams-with-unreadable-text-color/11155/4
https://forum.obsidian.md/t/text-colors-for-mermaid-mindmap-graph-are-unreadable-in-dark-mode/68674

Obviously colors and readability is a pain point in mermaid. The best option would probably be to create the title my markdown outside of mermaid. This would be consistent with other non-mermaid diagrams and (honestly speaking) I do not want to invest more hours and days to this topic.

**Comment by maturar** [comment]
All findings have been fixed. The review can continue.

File: doc/docs/reference/control-interface.md [hunk 1]
```diff
@@ -190,6 +177,8 @@ flowchart TD
     further_processing --> fin
 ```
 
+<figcaption>Read response message via control interface</figcaption>
```
Comment (windsource) [inline doc/docs/reference/control-interface.md:180]: I think `<figcaption>` is supposed to be used inside a `<figure>`element. As here it used standalone it is not centered. I have tried to add a `<figure>` element around mermaid and the caption but then the image rendered by mermaid gets very small.

So we have the following options:

1. Have a centered caption but very small images - Not good
2. Have no caption at all - Caption is helpful for some images
3. Replace mermaid with images that are rendered offline - Maybe something for the future.
4. Keep this PR as it is

From all options I think the last one is best for the time being

**Review by windsource — APPROVED** [review]
Ok with previous comment.

File: doc/docs/reference/control-interface.md (hunk 1 above)
Comment (maturar) [reply doc/docs/reference/control-interface.md:184]: I have added comments to the makrdown sources about it. Is it okay for you?

**Pull request merged**